client/actions/main.js

```diff
@@ -237,6 +237,8 @@ const save = (item, withConfirmation = true, noSubsequentEditing = false) => (
                             dispatch(planningApi.receivePlannings([planningUtils.modifyForClient(savedItem)]));
                             break;
                         }
+
+                        return Promise.resolve(savedItem);
```

Comment:
Returning here means the end user doesn't get the following message on save:
`The Event/Planning has been saved`